jdbc/basics/src/test/java/example/springdata/jdbc/basics/aggregate/AggregateTests.java

```diff
@@ -24,20 +24,21 @@
 import java.util.List;
 
 import org.assertj.core.groups.Tuple;
-import org.junit.Test;
-import org.junit.runner.RunWith;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.data.jdbc.AutoConfigureDataJdbc;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.annotation.DirtiesContext;
-import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
 
 /**
  * Demonstrates various possibilities to customize the behavior of a repository.
  *
  * @author Jens Schauder
  */
-@RunWith(SpringRunner.class)
+@ExtendWith(SpringExtension.class)
```

Comment:
Actually, `@ExtendWith` shouldn't be necessary at all since `@SpringBootTest` is already annotated with `@ExtendWith(SpringExtension.class)`

Reply:
done the changes.